File: test_debug_program.c
#include <stdio.h>

int main() {
    int counter = 0;

    printf("Starting debug test program...\n");

    // Breakpoint здесь (строка 8)
    counter = 42;

    printf("Counter value: %d\n", counter);

    // Еще один breakpoint (строка 12)
    counter = counter * 2;

    printf("Final counter: %d\n", counter);

    return 0;
}

File: test_debug_program.rs
fn main() {
    let mut counter = 0;

    println!("Starting debug test program...");

    // Breakpoint здесь (строка 6)
    counter = 42;

    println!("Counter value: {}", counter);

    // Еще один breakpoint (строка 10)
    counter = counter * 2;

    println!("Final counter: {}", counter);
}
